test/jdk/java/nio/file/Files/probeContentType/Basic.java

```diff
@@ -153,39 +156,55 @@ public static void main(String[] args) throws IOException {
-                new ExType("rtf", List.of("application/rtf", "text/rtf")),
-                new ExType("webm", List.of("video/webm")),
-                new ExType("webp", List.of("image/webp")),
-                new ExType("xls", List.of("application/vnd.ms-excel")),
-                new ExType("xlsx", List.of("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")),
-                new ExType("7z", List.of("application/x-7z-compressed")),
-        };
+        List<ExType> exTypes = new ArrayList<ExType>();
+
+        // extensions with consistent content type
+        exTypes.add(new ExType("adoc", List.of("text/plain")));
+        exTypes.add(new ExType("css", List.of("text/css")));
+        exTypes.add(new ExType("doc", List.of("application/msword")));
+        exTypes.add(new ExType("docx", List.of("application/vnd.openxmlformats-officedocument.wordprocessingml.document")));
+        exTypes.add(new ExType("gz", List.of("application/gzip", "application/x-gzip")));
+        exTypes.add(new ExType("jar", List.of("application/java-archive", "application/x-java-archive", "application/jar")));
+        exTypes.add(new ExType("jpg", List.of("image/jpeg")));
+        exTypes.add(new ExType("js", List.of("text/plain", "text/javascript", "application/javascript")));
+        exTypes.add(new ExType("json", List.of("application/json")));
+        exTypes.add(new ExType("markdown", List.of("text/markdown")));
+        exTypes.add(new ExType("md", List.of("text/markdown", "application/x-genesis-rom")));
+        exTypes.add(new ExType("mp3", List.of("audio/mpeg")));
+        exTypes.add(new ExType("mp4", List.of("video/mp4")));
+        exTypes.add(new ExType("odp", List.of("application/vnd.oasis.opendocument.presentation")));
+        exTypes.add(new ExType("ods", List.of("application/vnd.oasis.opendocument.spreadsheet")));
+        exTypes.add(new ExType("odt", List.of("application/vnd.oasis.opendocument.text")));
+        exTypes.add(new ExType("pdf", List.of("application/pdf")));
+        exTypes.add(new ExType("php", List.of("text/plain", "text/php", "application/x-php")));
+        exTypes.add(new ExType("png", List.of("image/png")));
+        exTypes.add(new ExType("ppt", List.of("application/vnd.ms-powerpoint")));
+        exTypes.add(new ExType("pptx", List.of("application/vnd.openxmlformats-officedocument.presentationml.presentation")));
+        exTypes.add(new ExType("py", List.of("text/plain", "text/x-python", "text/x-python-script")));
+        exTypes.add(new ExType("webm", List.of("video/webm")));
+        exTypes.add(new ExType("webp", List.of("image/webp")));
+        exTypes.add(new ExType("xls", List.of("application/vnd.ms-excel")));
+        exTypes.add(new ExType("xlsx", List.of("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")));
+        // exTypes.add(new ExType("wasm", List.of("application/wasm")));
```

Comment:
Ok, [JDK-8297609](https://bugs.openjdk.org/browse/JDK-8297609) is not in 17u. Can you add a comment explaining this, please?

Reply:
@TheRealMDoerr - Done